pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -351,7 +352,8 @@ func (k *Kubernetes) ConfigTmpfs(name string, service kobject.ServiceConfig) ([]
 	for index, volume := range service.TmpFs {
 		//naming volumes if multiple tmpfs are provided
 		volumeName := fmt.Sprintf("%s-tmpfs%d", name, index)
-
+		v := strings.Split(volume, ":")
+		volume = v[0]
```

Comment:
this could be just `volume = strings.Split(volume, ":")` remove the other line

Reply:
No, it will not work, 

```
# github.com/kubernetes/kompose/pkg/transformer/kubernetes
pkg/transformer/kubernetes/kubernetes.go:355: cannot use strings.Split(volume, ":") (type []string) as type string in assignment
```

Reply:
@surajnarwade Pretty easy to figure it out, I forgot to add the [0].
Do: `strings.Split(volume, ":")[0]`